index.html

```diff
@@ -1,8 +1,44 @@
 <html>
-  <head>
-    <title>Memory exercise</title>
-  </head>
-  <body>
-  </body>
+<head>
+<meta charset="UTF-8">
+<title>Game V2</title>
+<link href="main.css" rel="stylesheet" type="text/css">
+<link href='https://fonts.googleapis.com/css?family=Amatic+SC|Exo|Dancing+Script' rel='stylesheet' type='text/css'>
+<script src="http://code.jquery.com/jquery-1.11.3.min.js"></script
+<script src="game_work.js" ></script>
+</head>
+
+<body>
+ <div id='Header' style="width: 120px; height:auto;">
+    <p class = 'resetGame'> Reset </p> 
+ </div>
+ <div id='newNode'>
+   <p> New new node </p> 
+ </div>
+ <ul class="v2_center2">
+ </ul>
+ <ul class="v2_center"> <!-- image credit website https://www.iconfinder.com/iconsets/ballicons-free -->
+      <li><a href="#"><img class="hid0" src="assets/wine.png" alt="wine"><img src="assets/open-box.png" alt="open-box"></a></li>
+      <li data = "2"><a href="#"><img class="hid1" src="assets/oypad.png" alt="oypad"><img src="assets/open-box.png" alt="wine"></a></li> 
+      <li data = "3"><a href="#"><img class="hid2" src="assets/man.png" alt="man"><img src="assets/open-box.png" alt="wine"></a></li>
+      <li data = "4"><a href="#"><img class="hid3" src="assets/target.png" alt="target"><img src="assets/open-box.png" alt="wine"></a></li>  
+    
+      <li data = "2"><a href="#"><img class="hid4" src="assets/oypad.png" alt="oypad"><img src="assets/open-box.png" alt="wine"></a></li>
+      <li data = "4"><a href="#"><img class="hid5" src="assets/target.png" alt="target"><img src="assets/open-box.png" alt="wine"></a></li> 
+      <li data = "3"><a href="#"><img class="hid6" src="assets/man.png" alt="man"><img src="assets/open-box.png" alt="wine"></a></li>
+      <li data = "1"><a href="#"><img class="hid7" src="assets/wine.png" alt="wine"><img src="assets/open-box.png" alt="wine"></a></li> 
+
+      <li data = "2"><a href="#"><img class="hid8" src="assets/imac.png" alt="oypad"><img src="assets/open-box.png" alt="wine"></a></li>
+      <li data = "4"><a href="#"><img class="hid9" src="assets/keyboards.png" alt="target"><img src="assets/open-box.png" alt="wine"></a></li> 
+      <li data = "3"><a href="#"><img class="hid10" src="assets/diamond.png" alt="man"><img src="assets/open-box.png" alt="wine"></a></li>
+      <li data = "1"><a href="#"><img class="hid11" src="assets/speakers.png" alt="wine"><img src="assets/open-box.png" alt="wine"></a></li>
+
+      <li data = "2"><a href="#"><img class="hid12" src="assets/diamond.png" alt="oypad"><img src="assets/open-box.png" alt="wine"></a></li>
+      <li data = "4"><a href="#"><img class="hid13" src="assets/speakers.png" alt="target"><img src="assets/open-box.png" alt="wine"></a></li> 
+      <li data = "3"><a href="#"><img class="hid14" src="assets/keyboards.png" alt="man"><img src="assets/open-box.png" alt="wine"></a></li>
+      <li data = "1"><a href="#"><img class="hid15" src="assets/imac.png" alt="wine"><img src="assets/open-box.png" alt="wine"></a></li>
```

Comment:
I actually have an idea of using js generating the DOM. I was  manage to do so in the homework for ajax at jb012.com/mashup  that i am going to submit it very very soon.